android/src/androidTest/java/org/ligi/gobandroidhd/uitest/TheGameInfoDialog.java

```diff
@@ -66,22 +67,22 @@ public void testThatFieldsWork() {
         final GameReviewActivity activity = getActivity();
         onView(withId(R.id.menu_game_info)).perform(click());
 
-        onView(withId(R.id.game_name_et)).perform(typeText(CUSTOM_GAME_NAME));
+        onView(withId(R.id.game_name_et)).perform(replaceText(CUSTOM_GAME_NAME));
```

Comment:
This testcase can't pass on none english input method,  when call TypeText("custom black:),  I got  chinese  character,  I can't find another way to change default ime from chinese to english.   
I'm not familiar with test framework,  you can ignore this change if this is not correct.